src/main/drivers/accgyro/accgyro_spi_icm426xx.c

```diff
@@ -103,11 +96,45 @@
 #define ICM426XX_INT_TPULSE_DURATION_100            (0 << ICM426XX_INT_TPULSE_DURATION_BIT)
 #define ICM426XX_INT_TPULSE_DURATION_8              (1 << ICM426XX_INT_TPULSE_DURATION_BIT)
 
-
 #define ICM426XX_RA_INT_SOURCE0                     0x65
 #define ICM426XX_UI_DRDY_INT1_EN_DISABLED           (0 << 3)
 #define ICM426XX_UI_DRDY_INT1_EN_ENABLED            (1 << 3)
 
+typedef struct odrConfig_s {
+    uint8_t odr;            // ODR: Output data rate (gyro sample rate in kHz)
+    uint8_t odr_idx;        // See GYRO_ODR in datasheet
+} odrConfig_t;
+
+typedef struct aafConfig_s {
+    uint8_t delt;
+    uint16_t deltSqr;
+    uint8_t bitshift;
+} aafConfig_t;
+
+typedef enum {
+    AAF_CONFIG_258HZ = 0,
+    AAF_CONFIG_536HZ,
+    AAF_CONFIG_997HZ,
+    AAF_CONFIG_1962HZ,
+    AAF_CONFIG_COUNT
+} aafConfig_e;
+
+// Possible output data rates (ODRs)
+static odrConfig_t odrLUT[] = {  // see tables in section 5.6
+    { 8, 3 },  // 8 kHz
+    { 4, 4 },  // 4 kHz
+    { 2, 5 },  // 2 kHz
+    { 1, 6 },  // 1 kHz
+};
+
+// Possible gyro Anti-Alias Filter cutoffs
+static aafConfig_t aafLUT[AAF_CONFIG_COUNT] = {  // see table in section 5.3
+    {  6,   36, 10 },  //  258 Hz
```

Comment:
```suggestion
static aafConfig_t aafLUT[] = {
  [GYRO_HARDWARE_LPF_NORMAL] = {  6,   36, 10 },  //  258 Hz
 ...
```
```

Reply:
I defined `aafConfig_e` and explicitly mapped that to the `gyroHardwareLpf_e` elements on purpose to make it more robust against possible bugs. Just so that the order and content of `gyroHardwareLpf_e` is invariant against this implementation. Future people working on the code have one tripping line less to worry about.

Reply:
IMO both will do about the same, it's your call